pkg/controller/autoscaling/elasticsearch/resources/resources.go

```diff
@@ -15,8 +15,10 @@ import (
 )
 
 const (
-	// GIB - 1 GibiByte
+	// GIB - 1 GibiBytes
 	GIB = int64(1024 * 1024 * 1024)
```

Comment:
Nit: Unrelated to this PR but the acronym of Gibibyte is GiB with the i in lowercase.